primitives/core/src/offchain/mod.rs

```diff
@@ -495,6 +497,9 @@ pub trait Externalities: Send {
 		buffer: &mut [u8],
 		deadline: Option<Timestamp>
 	) -> Result<usize, HttpError>;
+
+	/// Set the reserved nodes
```

Comment:
This needs way more documentation than just `Set the reserved nodes`

- What is a reserved node?
- What is the reserved-only mode?
- What does this `Vec<Vec<u8>>` mean?
- What happens in case of a fork?